### PR TITLE
feat: Update favorite chips to use Button

### PR DESCRIPTION
**Pull request by cbrevik**
Also fix issues with FavoriteIcons fill color, and allow Button to control icon fill (shouldn't use ThemeIcon in that case).

Added FadeBetween to fade between two children. Also removed explicit marginBottom on Buttons.

### CODE REVIEW

File: src/components/search-button/index.tsx [hunk 1]
```diff
@@ -76,8 +76,7 @@ const useThemeStyles = StyleSheet.createThemeHook((theme) => ({
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
-    paddingHorizontal: theme.spacings.medium,
-    paddingVertical: theme.spacings.small,
+    padding: theme.spacings.medium,
```
Comment (rikkekantega) [inline src/components/search-button/index.tsx:79]: Just commenting on this: Changed to minHeight on the other branch, but both increased padding and defined minHeight might as well live together :)

**Review by rikkekantega — COMMENTED** [review]
image
The two rows of favoritechips should be provided some spacing between :thinking:

**Comment by rikkekantega** [comment]
image

Favorites are now being capped on the right end side, not sure if intentional

**Comment by cbrevik** [comment]
Oops, will fix both issues! My last commit messed it up a bit

**Comment by cbrevik** [comment]
Should be fixed 👍 @rikkekantega Good catch 😅

**Review by rikkekantega — APPROVED** [review]
I love this! :grinning:  LGTM+ :star:

**Pull request merged**